src/games/strategy/util/PointFileReaderWriter.java

```diff
@@ -149,6 +149,31 @@ public static void writeOneToMany(final OutputStream sink, Map mapping) throws E
     }
     write(out, sink);
   }
+  
+  /*public static void writeOneToMany(final OutputStream sink, Map<String, List<Point>> mapping) throws Exception {
```

Comment:
Sorry, I tested, if java differs between Map<String, List<Point>> and Map<String, Collection<Point>> it does not...